app/instrumentation-tests/src/org/commcare/androidTests/ManualQuarantineTest.kt

```diff
@@ -0,0 +1,126 @@
+package org.commcare.androidTests
+
+import android.os.Build
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.action.ViewActions.longClick
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import androidx.test.filters.SdkSuppress
+import org.commcare.dalvik.R
+import org.commcare.utils.CustomMatchers
+import org.commcare.utils.InstrumentationUtility
+import org.commcare.utils.doesNotExist
+import org.commcare.utils.isDisplayed
+import org.hamcrest.Matchers.allOf
+import org.junit.Before
+import org.junit.FixMethodOrder
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.MethodSorters
+
+@RunWith(AndroidJUnit4::class)
+@LargeTest
+@SdkSuppress(maxSdkVersion = Build.VERSION_CODES.Q)
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+class ManualQuarantineTest: BaseTest() {
+
+    companion object {
+        const val CCZ_NAME = "ccqa.ccz"
+        const val APP_NAME = "Basic Tests"
+    }
+
+    @Before
+    fun setup() {
+        installApp(APP_NAME, CCZ_NAME)
+        InstrumentationUtility.login("test", "1234")
+        // Enable quarantine
+        enableFormQuarantine()
+        InstrumentationUtility.changeWifi(false)
+    }
+
+    private fun enableFormQuarantine() {
+        InstrumentationUtility.gotoHome()
+        InstrumentationUtility.selectOptionItem(withText("Advanced"))
+        onView(withText("Enable Manual Form Quarantine"))
+                .perform(click())
+        InstrumentationUtility.gotoHome()
+    }
+
+    @Test
+    fun test_A_Quarantine() {
+        InstrumentationUtility.openModule(12)
```

Comment:
nit: if we are using position as input to `openModule`, we should put a comment here specifiying the text of module we want to open for better readability or move the value `12` in a constant whose name indicates what module this position is for.